numpy/core/tests/test_multiarray.py

```diff
@@ -92,6 +92,44 @@ def test_writeable(self):
         self.a[0] = 5
         self.a[0] = 0
 
+        # gh-9440 - make sure fromstring, from buffer on readonly buffers
```

Comment:
Can you add `def test_writeable_from_readonly` here to split the tests up a little?

Reply:
done